.circleci/config.yml

```diff
@@ -146,7 +147,7 @@ jobs:
           py_version: << parameters.py_version >>
       - run:
           name: Testing Hydra
-          no_output_timeout: 10m
```

Comment:
This timeout is on the tests. Didn't we see the timeout while preparing the environment?

Reply:
🤦🏻‍♀️  will fix